src/app/tests/suites/commands/interaction_model/InteractionModel.h

```diff
@@ -258,7 +258,7 @@ class InteractionModelCommands
 
             if (mRepeatDelayInMs.HasValue())
             {
-                chip::test_utils::SleepMillis(mRepeatDelayInMs.Value());
```

Comment:
Please leave `test_utils::` namespace.

We don't want to start seeing standard non-test-code call things like "SleepMillis" here and there.

Reply:
Ok, I've reverted my changes and left only one related to breaking dependencies.